kubetest/README.md

```diff
@@ -177,3 +180,17 @@ skew directory, potentially to upgrade/downgrade kubernetes to another version.
 [kubekins-e2e]: /jenkins/e2e-image
 [kubekins-e2e-prow]: /images/e2e-prow
 [prow]: /prow
+
+## Example Commands
```

Comment:
Move this up from the links, maybe this can be a good example for `### Upgrade, skew, kubemark, federation` section.

Reply:
sure!